WcaOnRails/app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,8 @@
   <link rel="icon" href='<%= asset_path("android-touch-icon.png") %>'/>
   <link rel="apple-touch-icon" href='<%= asset_path("apple-touch-icon.png") %>'/>
 
+  <meta name="og:image" content="https://www.worldcubeassociation.org/assets/og-wca_logo.png">
```

Comment:
Please use `asset_path` here instead (or `asset_url` if it must be a full url). It's silly for people developing locally to be pointing at our production server.

Reply:
Ok! I'll make that change right now.